src/styles/mixins/_typography.scss

```
@@ -5,6 +5,10 @@
     font-family: Lato, 'Helvetica Neue', Helvetica, Arial, sans-serif;
 }
 
+@mixin bdl-fontFamily--inter {
```

Stylelint (17.14.0)

[error] Expected mixin name to be kebab-case (scss/at-mixin-pattern)

(scss/at-mixin-pattern)